pkg/block/adapter.go

```diff
@@ -15,6 +15,10 @@ type MultipartUploadCompletion struct{ Part []*s3.CompletedPart }
 type ObjectPointer struct {
 	StorageNamespace string
 	Identifier       string
+
+	// Indicates whether the Identifier is relative to the StorageNamespace.
+	// nil value means it's unknown.
```

Comment:
Easier to read _and_ write calls if we use a tristate bool here.

Reply:
Done.